README.md

```diff
@@ -166,9 +166,24 @@ spec:
 | Private Builder Image Registry     | ⚪️     |      |    |
 | Runtime Base Image    | ⚪️    |      |    |
 | Binary builds         |       |      |    |
+| Image pull policy     |       |      |    |
+| Inject source via config map         |       |      |    |
+| Inject source via secret         |       |      |    |
+| Inject source via image         |       |      |    |
+| Set image label/ens   |       |      |    |
+| Triggers (SCMs)       |       |      |    |
+| Triggers (imagestream)|       |      |    |
+| Pruning of builds     |       |      |    |
+| Cancelling of builds  |       |      |    |
+| Chaining of builds    |       |      |    |
```

Comment:
doable via tekton today .... just whether buildv2 encapsulates this in the API like buildv1 did

Reply:
buildv1 didn't really encapsulate it.  it just allowed input content to come from an existing image, which you've already covered, and for the push of one image to trigger the build of another, which you've arguably already covered also w/ imagestream triggers

Reply:
ah that's right ... thanks for the clarification.  I'll remove this from this table, or a subsequent candidates table if we go down that path, and will refrain from opening an issue for discussion, per that last ask from @siamaksade